.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
```

Comment:
Maybe restrict this only to the `master` branch?

Reply:
Makes sense, I’ll update this along with the previous comment’s change

Reply:
I tried with it and the build wouldn't trigger anymore in my fork's branch after push so there's no way to test changes to gh actions configuration.